src/oemof/tabular/_facade.py

```diff
@@ -151,7 +151,7 @@ def __init__(self, *args, **kwargs):
             add_subnodes, sender=self
         )
 
-    def _nominal_value(self):
+    def _nominal_value(self, value=None):
```

Comment:
Why do you need this?
Can't you just use whatever value you need directly without this function?

Reply:
As discussed, I will stick to this, as it will not break the status quo and reduced code duplication especially in the BEV facade